src/js/profile/wearable/widget/wearable/NumberPicker.js

```diff
@@ -46,7 +46,7 @@
 				selectors = ns.util.selectors,
 				prototype = new ns.widget.BaseWidget(),
 				Spin = ns.widget.wearable.Spin,
-				SPIN_CLASS = "ui-spin",
+				SPIN_CLASS = Spin.classes.widget,
```

Comment:
Is the '.widget' corect value in the 'Spin.classes'?

Reply:
You are right. Incorrect.

Reply:
Fixed.